language/move-lang/tests/move_check/expansion/restricted_address_names.exp

```diff
@@ -1,12 +1,12 @@
-error[E02010]: invalid name
+error[E03011]: invalid use of reserved name
   ┌─ tests/move_check/expansion/restricted_address_names.move:1:8
   │
 1 │ module Self::A {}
-  │        ^^^^ Invalid named address 'Self'. 'Self' is restricted and cannot be used to name an address
+  │        ^^^^ Invalid address name 'Self'. 'Self' is restricted and cannot be used to name a address
```

Comment:
This should be "an address"

Reply:
oof testing my rust skills let's see how this works lol